crates/ruff_python_index/src/multiline_ranges.rs

```diff
@@ -9,7 +9,7 @@ pub struct MultilineRanges {
 
 impl MultilineRanges {
     /// Returns `true` if the given range is inside a multiline string.
-    pub fn intersects(&self, target: TextRange) -> bool {
+    pub fn contains(&self, target: TextRange) -> bool {
```

Comment:
I think this was a confusing name. `contains` indicates that the query is: "Does this `MultilineRanges` contain the `TextRange`?"

Reply:
But all the rest are still `intersects`?

Reply:
Sorry, say more?

Reply:
Why is this method changing from `intersects` to `contains` but there are a bunch of other methods with the same signature that are called `intersects`? I don't quite get it.

Reply:
Those are entirely new methods with different behavior. The docstrings and implementations are different. (`intersects` calls `target.contains_range(*range)`, while this method calls `range.contains_range(target)`.)

Reply:
Thanks for clarifying. I guess my point is we have

- `X.contains(Y)` meaning that `Y` is inside of `X` 
- `X.intersects(Y)` meaning that `X` is inside of `Y`

I think the first is relatively obvious but the second is less so. What about `within`?

Reply:
That's not quite what they mean but of course open to ways to make them clearer. `X` is a collection of ranges, and `Y` is a single range. Thus:

- `X.contains(Y)` means that `Y` is contained within an entry in `X`.
- `X.intersects(Y)` means that `Y` intersects with an entry in `X`.

Reply:
Oh I see... `covers` seems a bit clearer. I think I'd have to actually play with this API to have a strong opinion though, so definitely do what you think is best. I'm curious to hear what other people think too.

Reply:
(It might also be clearer if I change the check _within_ `intersects` to `target.intersects(*range)`. It doesn't make a difference in practice, since the `target` can never intersect-but-not-contain a string.)

Reply:
I expect the method to return `true` if `target` intersects with any multline string. But that's not what the method does. It only returns `true` if `target` is fully contained by one of the multline string ranges. 

Now, the solution is to either:
* change the method to return whether `target` intersects with any multline string and keep the name as is
* Change the name to `contains` if that's the semantic we want.

My general advice here would be that we design the naming similar to the `TextRange` methods to use a single terminology when it comes to ranges.

Edit: That's what @charliermarsh's last comment says too :)